CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 - Removed unused table `bdms.statigraphy_codelist` from database.
 - Removed  `layer.unconrocks_id_cli` and `layer.lithok_id_cli` from database.
 - Changed cardinality of `uscs3` field on `bdms.layer` to m:n.
+- Get Municipality, Canton, Country from geo.admin.ch service. Removed obsolete Canton / Municipality tables.
```

Comment:
REQ: Das müsste in die unreleased section des changelog. Da es inzwischen einen Release gab.